__test__/event-feed.test.ts

```diff
@@ -0,0 +1,176 @@
+import eventFeedHandler from "../pages/api/event-feed";
+import { client } from "../lib/db";
+import { makeHTTPRequest, makeEventFeedHTTPRequest } from "./__testutils__/testutils.test";
+import { CalendarEvent } from "../models/events";
+import { StatusCodes } from "http-status-codes";
+
+const MISSING_PARAMS_ERROR = "Missing required parameters";
+
+beforeAll(async () => {
+  await client.query("DELETE from event_information");
+  await client.query("DELETE from calendar_information");
+  await client.query("DELETE from commitments");
+  await client.query("DELETE from users");
+  await client.query(
+    "INSERT INTO event_information(id, name, background_color, type, never_ending) VALUES('id_a', 'event_a', 'blue', 'Class', false)"
+  );
+  await client.query(
+    "INSERT INTO event_information(id, name, background_color, type, never_ending) VALUES('id_b', 'event_b', 'red', 'Class', false)"
+  );
+  await client.query(
+    "INSERT INTO event_information(id, name, background_color, type, never_ending) VALUES('id_c', 'event_c', 'green', 'Class', false)"
+  );
+  await client.query(
+    "INSERT INTO calendar_information(event_information_id, start_str, end_str) VALUES('id_a', '2022-02-26 21:00:00-08', '2022-02-26 21:00:00-08')"
+  );
+  await client.query(
+    "INSERT INTO calendar_information(event_information_id, start_str, end_str) VALUES('id_a', '2022-02-27 21:11:45-08', '2022-02-27 21:11:45-08')"
+  );
+  await client.query(
+    "INSERT INTO calendar_information(event_information_id, start_str, end_str) VALUES('id_b', '2022-02-24 21:11:45-08', '2022-02-24 21:11:45-08')"
+  );
+  await client.query(
+    "INSERT INTO calendar_information(event_information_id, start_str, end_str) VALUES('id_b', '2022-02-26 21:11:45-08', '2022-02-26 21:11:45-08')"
+  );
+  await client.query(
+    "INSERT INTO calendar_information(event_information_id, start_str, end_str) VALUES('id_c', '2022-02-23 21:11:45-08', '2022-02-23 21:11:45-08')"
+  );
+  await client.query(
+    "INSERT INTO calendar_information(event_information_id, start_str, end_str) VALUES('id_c', '2022-03-01 21:11:45-08', '2022-03-01 21:11:45-08')"
+  );
+  await client.query(
+    "INSERT INTO users(id, email, role, first_name, last_name, phone_number, address) VALUES('user_a', 'emaila@gmail.com', 'role', 'fname', 'lname', '#', 'addr')"
+  );
+  await client.query(
+    "INSERT INTO commitments(user_id, event_information_id) VALUES('user_a', 'id_a')"
+  );
+});
+
+afterAll(async () => {
+  await client.end();
+});
+
+describe("[GET] /api/event-feed", () => {
+  test("get calendar event feed for a user", async () => {
+    const body = {
+      start: "2022-02-25 21:11:45-08",
+      end: "2022-02-28 21:11:45-08",
+      userId: "user_a",
+    };
```

Comment:
Oof, im sorry if i messed up giving this information out. The API takes in nothing in the body. Everything is done via query parameters. So everything is the same, except these should be passed in as the query parameter of  the http request